modules/xml/xml-grammar.ym

```diff
@@ -84,8 +84,10 @@ xml_options
 xml_opt
         : KW_PREFIX '(' string ')'      { xml_parser_set_prefix(last_parser, $3); free($3); }
         | KW_DROP_INVALID '(' yesno ')' { xml_parser_set_forward_invalid(last_parser, !$3); };
-        | KW_EXCLUDE_TAGS '(' string_list ')' { xml_parser_set_exclude_tags(last_parser, $3); g_list_free_full($3, free); }
-        | KW_STRIP_WHITESPACES '(' yesno ')' { xml_parser_set_strip_whitespaces(last_parser, $3); }
+        | KW_EXCLUDE_TAGS '(' string_list ')'
+          { xml_scanner_options_set_exclude_tags(xml_parser_get_scanner_options(last_parser), $3); g_list_free_full($3, free); }
```

Comment:
Refactor tip:
Declare last_xml_scanner_options and call get_scanner_options only once in `KW_XML`.

Reply:
It's nicer that way, I can add that.